api/src/main/java/com/epam/esm/config/DatabaseConfig.java

```diff
@@ -0,0 +1,57 @@
+package com.epam.esm.config;
+
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.datasource.DataSourceTransactionManager;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+import javax.sql.DataSource;
+
+@Configuration
+@PropertySource(value = "classpath:application-${spring.profiles.active}.properties", ignoreResourceNotFound = true)
+@EnableTransactionManagement
+@Profile({"prod", "dev"})
+public class DatabaseConfig {
```

Comment:
Learn how in spring boot configure datasource- remove this class. 
 spring.datasource.url=
spring.datasource.username=
spring.datasource.password=
spring.datasource.driver-class-name=